changelog.mdx

```diff
@@ -4,6 +4,45 @@ description: "All the nice stuff you asked for"
 icon: "sparkles"
 ---
 
+## 2024-02-23
+
+- `improved stability`: (not sure how to explain here)
+- `discover test cases`: our AI Agent will analyse your webpage and suggest multiple prompts which describe your test flows
+
+<Frame caption="screenshot 02/2024">
+  <img src="images/changelog/discover-screen.png" alt="new discovery feature" />
+</Frame>
+
+- `approval workflow`: once our AI Agent has finished the discovery, you have an option to approve or discard provided suggestions.
+  Discarding will delete the draft, while approval will give the prompt to our AI Agent to start generating steps from it
+
+<Frame caption="screenshot 02/2024">
+  <img src="images/changelog/approval-screen.png" alt="new discovery feature" />
+</Frame>
+
+## 2024-02-15
+
+- `test report generation speed up` : we are now running published test cases within a report in parallel
+- `auto fix`: you can now select which test cases our AI Agent should try to auto fix once they fail
+
+<Frame caption="screenshot 02/2024">
+  <img src="images/changelog/auto-fix.png" alt="auto fix feature" />
+</Frame>
+
+- save rail overhaul:
+
+  - we replaced save rail with the functionalities that were inside it. Now we have 3 separate buttons
```

Comment:
```suggestion
  we replaced save rail with the functionalities that were inside it. Now we have 3 separate buttons
```

Reply:
maybe look at that one, but we don't need single bullet point within single bulelt point